### PR TITLE
S04 us84 slack button on idea pages

### LINKED ISSUE #84: En tant qu'auteur de l'idée, je vois un bouton Slack stylisé pour créer ce canal sur chaque page idée

### PR DESCRIPTION
**Pull request by AurelOconnell**

### CODE REVIEW

**Review by Spolito24 — APPROVED** [review]

**Review by benrcd — CHANGES_REQUESTED** [review]
style="
                            align-items: center;
                            color: #fff;
                            background-color: #4a154b;
                            border: 0;
                            border-radius: 4px;
                            display: inline-flex;
                            font-family: Lato, sans-serif;
                            font-size: 16px;
                            font-weight: 600;
                            height: 48px;
                            justify-content: center;
                            text-decoration: none;
                            width: 100%;
                        "

The CSS should be in the SCSS file

**Comment by AurelOconnell** [comment]
Need to show this button for author's idea only :)

**Review by Spolito24 — APPROVED** [review]

File: ngrok.exe
Comment (aldaitzwild) [inline ngrok.exe:1]: No .exe should be versionned in git. It's not a part of your code

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: ngrok.exe
Comment (aldaitzwild) [reply ngrok.exe:1]: Still in the PR

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**